src/Telegram.Bot/TelegramBotClient.cs

```diff
@@ -763,13 +763,23 @@ public Task<User> GetMeAsync(CancellationToken cancellationToken = default)
             bool disableNotification = default,
             int replyToMessageId = default,
             IReplyMarkup replyMarkup = default,
-            CancellationToken cancellationToken = default
```

Comment:
I'd suggest to mark this as an [Obsolete] and add overload with correct parameters order, so this one could be removed upon next release.